src/coprocessor/codec/batch/lazy_column_vec.rs

```diff
@@ -35,7 +35,29 @@ impl From<Vec<LazyBatchColumn>> for LazyBatchColumnVec {
     }
 }
 
+impl From<Vec<VectorValue>> for LazyBatchColumnVec {
+    #[inline]
+    fn from(columns: Vec<VectorValue>) -> Self {
+        LazyBatchColumnVec {
+            columns: columns
+                .into_iter()
+                .map(|v| LazyBatchColumn::from(v))
+                .collect(),
+        }
+    }
+}
+
 impl LazyBatchColumnVec {
+    /// Creates a new empty `LazyBatchColumnVec`, which does not have columns and rows.
+    ///
+    /// Because column numbers won't change, it means constructed instance will be always empty.
+    #[inline]
+    pub fn new_empty() -> Self {
```

Comment:
I think this function name can be similar to `raw`? Maybe change one of them?
BTW, is it good to combine these two?

Reply:
It is simply neither raw nor decoded, because there is even no column inside.